tasks/collect-collectors-data/README.md

```diff
@@ -0,0 +1,21 @@
+# collect-collectors-data
+
+Tekton task to collect the information related to collectors.
+
+The purpose of this task is to collect all the data and supply it to the other task in the pipeline by creating
+json files for the different resources.
+
+A task result is returned for each resource with the relative path to the stored JSON for it in the workspace. There is
+also a task result for the fbcFragment extracted from the snapshot's first component.
```

Comment:
fbcFragment? I don't see anything like that. Also, I don't really get it. I thought this would fetch "collectors" (whatever that is), but I am seeing it fetches the R, RP, RPA, just like the `collect-data` task. What's the difference then? Is this just a very similar task (sans data fields), but this one is gonna be used in some new pipeline?

Reply:
The difference is that this collects a subset of the resources. Only those associated with collectors.